pyproject.toml

```diff
@@ -138,6 +138,8 @@ filterwarnings = [
     "ignore::DeprecationWarning:glue",
     "ignore::DeprecationWarning:asteval",
     "ignore:::specutils.spectra.spectrum1d",
+    # Remove the following line once https://github.com/astrofrog/mpl-scatter-density/issues/46 is addressed
+    "ignore:pkg_resources is deprecated as an API:DeprecationWarning:mpl_scatter_density",
```

Comment:
So we can have reminder, can use a cross link here:

* https://github.com/astrofrog/mpl-scatter-density/issues/46